pkg/controllers/scheduling/scheduling_controller.go

```diff
@@ -147,28 +150,54 @@ func NewSchedulingController(
 		}, placementDecisionInformer.Informer()).
 		WithBareInformers(clusterInformer.Informer(), clusterSetInformer.Informer(), clusterSetBindingInformer.Informer()).
 		WithSync(c.sync).
+		ResyncEvery(ResyncInterval).
 		ToController(schedulingControllerName, recorder)
 }
 
 func (c *schedulingController) sync(ctx context.Context, syncCtx factory.SyncContext) error {
 	queueKey := syncCtx.QueueKey()
-	namespace, name, err := cache.SplitMetaNamespaceKey(queueKey)
-	if err != nil {
-		// ignore placement whose key is not in format: namespace/name
-		utilruntime.HandleError(err)
-		return nil
-	}
-
 	klog.V(4).Infof("Reconciling placement %q", queueKey)
-	placement, err := c.placementLister.Placements(namespace).Get(name)
-	if errors.IsNotFound(err) {
-		// no work if placement is deleted
+
+	if queueKey == "key" {
+		placements, err := c.placementLister.List(labels.Everything())
+		if err != nil {
+			return err
+		}
+
+		for _, placement := range placements {
+			for _, config := range placement.Spec.PrioritizerPolicy.Configurations {
+				if config.ScoreCoordinate != nil && config.ScoreCoordinate.Type == clusterapiv1alpha1.ScoreCoordinateTypeAddOn {
```

Comment:
I wonder do we need to resync for placement with customized addon only or we do not need this check

Reply:
In functionally, whether resync those placements that only uses "Steady", "Balance" or "ResourceAllocatableXXX" or not, the placement decision should be the same. 

A case I can think to resync all is, suppose a user has 2 placements, placement1 uses "Balance", placement2 uses "Balance" + "AddOn", when there are no changes in any CRs, the 2 placements will have different behavior, sounds strange and complex to understand.

Reply:
@elgnay any comments?